build_files/Rakefile

```diff
@@ -206,7 +206,7 @@ end
 
 def apply_bootstrap
   cputs "Running puppet apply on #{VM_TYPE}.pp"
-  system("PATH=$PATH:/opt/puppetlabs/puppet/bin puppet apply --modulepath=./modules:/opt/puppetlabs/puppet/modules --verbose ./manifests/#{VM_TYPE}.pp")
+  system("PATH=$PATH:/opt/puppetlabs/puppet/bin puppet apply --detailed-exitcodes --modulepath=./modules:/opt/puppetlabs/puppet/modules --verbose ./manifests/#{VM_TYPE}.pp")
```

Comment:
Exitcode 2 is actually "success with some changed resources". Are you checking to make sure this situation (others?) is handled well by packer?